learn/hoon/hoon-tutorial/type-checking-and-type-inference.md

```diff
@@ -190,13 +190,11 @@ $(a +(a), b (dec b))                                    ::  add a+1 and b-1
 We run it in the dojo using a cell to pass the two arguments:
 
 ```
-> +add [12 14]
+> (add 12 14)
```

Comment:
This is wrong, changes it from a generator invocation to just calling the gate

Reply:
`+add [12 14]` gives the error
```
+add [12 14]
%cx /~bus/home/~2019.9.6..20.58.49..61af/gen/add/hoon
scry failed for
ford: %hood failed for /~bus/home/0/gen/add/hoon:
ford: %core on /~bus/home/0/gen/add/hoon failed:
```

So I'm not sure how to make this a generator invocation that dojo likes